editor/js/libs/ui.js

```diff
@@ -534,6 +534,25 @@ UI.Color.prototype.setHexValue = function ( hex ) {
 
 // Number
 
+UI.nudgeNumericValue = function ( originalValue, keyCode, shiftKeyDown ) {
```

Comment:
I'd like to make this function private to the UI module, but I'm not sure how to do that. Suggestions are welcome!

Reply:
`nudgeNumericValue` should start with an uppercase letter.

Reply:
Have you considered to derive from `UI.Element.call( this );`?